tests/utils.go

```diff
@@ -5032,6 +5035,6 @@ func IsLauncherCapabilityValid(capability k8sv1.Capability) bool {
 	return false
 }
 
-func RetValue(retcode, prompt string) string {
-	return "\n" + retcode + "\r\n" + ".*" + prompt
+func RetValue(retcode string) string {
+	return "\n" + retcode + TermNLExpression + ".*" + PromptExpression
```

Comment:
dont you want (?s) in the `.*` ?

Reply:
This is not related to this change.
But it is also incorrect to use "?s" because the ".*" is there to "catch" various prompts prefixes (e.g. "[localhost]") and nothing else.

Reply:
if there is noise, noise can be with \n anyway, but up to you
(unless there isnt noise at , from any kind)

Reply:
I do not understand, what noise?
What is up to me? It is your comment, so please explain to what scenario you are pointing out here by giving an example.

Reply:
by noise i mean any stderr or prints that can burst in , the latter might not exists because `dmesg -n 1`
`.*` can catch the first one, and if there is a noise, noise can be with \n as well

i meant up to you because i dont find it important at this point to venture deeply in this.

Reply:
The value in this method should be the full output of the command. Another method can be introduced to support matching any sub string in the output. Anyway, not in the scope of this PR.

Reply:
sure its not needed to be in this scope, just raised that it might match noise, and if so it might to need to match also \n noise, or not match noise at all

Reply:
> by noise i mean any stderr or prints that can burst in , the latter might not exists because `dmesg -n 1`
> `.*` can catch the first one, and if there is a noise, noise can be with \n as well

Bursts that can get in are a general problem and allowing newline will not help as it can enter randomly all over, even between chars. And yes, it is handled by the dmesg setting.

Stderr will fail the expecter or not affect it, both are good.
In any case, we expect that after the returned value, there will be a \r\n with some string of the prompt that ends with either # or $. If instead there is a newline there it means it is not what we expected and should not match it.

If you have an explicit scenario where you think it still fails to do the expected, please share. If the scenario is indeed valid, we need to handle it in following pr/s.